fem/tbilininteg.hpp

```diff
@@ -554,6 +554,1126 @@ struct TDiffusionKernel<3,3,complex_t>
    }
 };
 
+// Mass kernel for Nedelec Finite Element
+
+// complex_t - type for the assembled data
+template <int SDim, int Dim, typename complex_t>
+struct THcurlMassKernel;
```

Comment:
Is there a reason why you need a new mass Kernel? Nedelec Finite Element are just a different basis, the standard mass Kernel should work no?

Reply:
The mapping for H1, H(curl), and H(div) are different. But we can redesign and simplify the mapping, like diffusion kernel in H1 are the same with the mass kernel in H(curl).